Makefile

```diff
@@ -42,8 +42,14 @@ else
 $(OUT_DIR)/logs/$(1)/$(2).log : RUNNER = ./tools/runner
 endif
 
+ifneq ($(RUNNER_KEEP_TMP),)
+RUNNER_PARAM := --keep-tmp
+else
+RUNNER_PARAM := --quiet
+endif
```

Comment:
Are these two mutually exclusive?

Reply:
To know which directory belongs to which test you need info logged by runner script https://github.com/SymbiFlow/sv-tests/pull/595/files#diff-3ff66653bd0ebf86aca23ca3ef75b58bR197
So you can use both options but that wouldn't be useful

Reply:
That makes sense, thanks for the explanation.